config/initializers/rollbar.rb

```diff
@@ -0,0 +1,53 @@
+Rollbar.configure do |config|
+  config.access_token = Rails.application.secrets.rollbar_token
+
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  config.enabled = Rails.env.production? && config.access_token.present?
+
+  # By default, Rollbar will try to call the `current_user` controller method
+  # to fetch the logged-in user object, and then call that object's `id`,
+  # `username`, and `email` methods to fetch those properties. To customize:
+  config.person_method = nil
```

Comment:
Out of an abundance of paranoia/cutting off any possible source of objection from higher up, let's take it a step further and _hash_ the id, and just send the hash. We'll still be able to tell if an issue is affecting one user or many, because different users will have different hashes and any one user will work out to the same hash, but a theoretically attacker can't use the Rollbar data to even infer user ids or relative age of accounts off of sequential id information